tokio/src/io/driver/scheduled_io.rs

```diff
@@ -56,6 +56,7 @@ impl Default for ScheduledIo {
 }
 
 impl ScheduledIo {
+    #[allow(dead_code)]
```

Comment:
why is this needed now?

Reply:
Because `get_readiness` is now only called from test code, so it would issue a warning. Could make it `#[cfg(test)]`, but then various _other_ methods become unused since they were only called from `get_readiness`. So instead of going through and marking a bunch of methods as `#[cfg(test)]`, I just made this `#[allow(dead_code)]`.

Reply:
I would prefer `#cfg(test)` we could move all of those fns to a separate impl block and annotate that block as cfg(test). Allowing dead_code will probably in that code eventually being unused and never deleted.

Reply:
Done